esphomeyaml/guides/faq.rst

```diff
@@ -197,6 +197,48 @@ Devices that are technically already supported, but for which guides will be cre
 Anything missing? I'd be happy to chat about more integrations over on the `discord channel
 <https://discord.gg/KhAMKrd>`__ - no guarantees that everything will be supported though!
 
+I can't update using OTA because of to little space, now what?
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+If you are using ESP8266/Sonoff devices you will probably encounter this error during OTA update:
```

Comment:
I don't want to scare off users 😅 Maybe add `if you have many features enabled`.